modules/nw-metallb-addresspool-cr.adoc

```diff
@@ -3,12 +3,12 @@
 // Module included in the following assemblies:
 //
 // * networking/metallb/metallb-configure-address-pools.adoc
-[id="nw-metallb-addresspool-cr_{context}"]
-= About the address pool custom resource
+[id="nw-metallb-ipaddresspool-cr_{context}"]
```

Comment:
As the addresspool CRD and API is still available in 4.11, would it make sense to make a note of it being deprecated here?

Reply:
hi @asood-rh I added a note in the Release notes http://file.emea.redhat.com/kquinn/TELCODOCS-506-411-RN/release_notes/ocp-4-11-release-notes.html#ocp-4-11-new-metallb-CRDs-logging-control. Maybe you want a note here too. I don't think it is deprecated as it is still supported as a way to configure AddressPools but doing it this way does not allow for the extended capabilities available in 4.11 - I think this is correct @fedepaol

Reply:
OK I think some note would be appropriate then as that message may cause concern